pkgs/misc/emulators/retroarch/0001-Disable-menu_show_core_updater.patch

```diff
@@ -0,0 +1,25 @@
+From 546b343294209abbb193883ab76b679b7f99c6d3 Mon Sep 17 00:00:00 2001
+From: Thiago Kenji Okada <thiagokokada@gmail.com>
+Date: Sat, 20 Nov 2021 16:03:50 -0300
+Subject: [PATCH 1/2] Disable "menu_show_core_updater"
+
+---
+ retroarch.cfg | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/retroarch.cfg b/retroarch.cfg
+index cdcb199c9f..ab72f3920f 100644
+--- a/retroarch.cfg
++++ b/retroarch.cfg
+@@ -681,7 +681,7 @@
+ # menu_show_online_updater = true
+ 
+ # If disabled, will hide the ability to update cores (and core info files) inside the menu.
+-# menu_show_core_updater = true
++menu_show_core_updater = false
```

Comment:
Does this disable manually installed cores too? I don’t know if anyone actually does this but I thought in the past you could imperitively install cores in the UI. If this can be enabled manually in the UI that would be alright.

Reply:
No, it just disable the UI to download new cores, that didn't work anyway since it doesn't connect to the RetroArch core servers (it would just show the cores that you already have installed). I don't know if this used to work, but at least the situation from version 1.8.5 (the version that was on `master` before I started updating and refactor everything) was already like this.

You can still re-enable this manually though, and this is respected on the next boot. But it will be useless.

Reply:
BTW, even if this used to work I don't think download would do anything.

First because this is NixOS, and thanks to this a core downloaded from RetroArch servers would probably just segfault (each core is a `.so` file and are dynamically linked, however since NixOS has no FHS it wouldn't find the necessary libraries).

Second because even before this PR, `libretro_directory` was already set to a path on `/nix/store`, that is read-only, so it would fail to write the core there. I am just setting `libretro_directory` directly in the code for consistency, but it was already the default.